cloud/ovh/ovh_ip_loadbalancing_backend.py

```diff
@@ -0,0 +1,298 @@
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+import sys
+try:
+    import ovh
+    import ovh.exceptions
+    from ovh.exceptions import APIError
+    HAS_OVH = True
+except ImportError:
+    HAS_OVH = False
+
+# import module snippets
+from ansible.module_utils.basic import *
+
+DOCUMENTATION = '''
+---
+module: ovh_ip_loadbalancing_backend
+short_description: Manage OVH IP LoadBalancing backends
+description:
+    - Manage OVH (French European hosting provider) LoadBalancing IP backends
+version_added: "2.0"
+author: Pascal HERAUD @pascalheraud
+notes:
+    - Uses the python OVH Api U(https://github.com/ovh/python-ovh). \
+    You have to create an application (a key and secret) with a consummer \
+    key as described into U(https://eu.api.ovh.com/g934.first_step_with_api)
+requirements:
+    - ovh >  0.3.5
+options:
+    name:
+        required: true
+        description:
+            - Name of the LoadBalancing internal name (ip-X.X.X.X)
+    backend:
+        required: true
+        description:
+            - The IP address of the backend to update / modify / delete
+    state:
+        required: false
+        default: present
+        choices: ['present', 'absent']
+        description:
+            - Determines wether the backend is to be created/modified \
+            or deleted
+    probe:
+        required: false
+        default: none
+        choices: ['none', 'http', 'icmp' , 'oco']
+        description:
+            - Determines the type of probe to use for this backend
+    weight:
+        required: false
+        default: 8
+        description:
+            - Determines the weight for this backend
+    endpoint:
+        required: true
+        description:
+            - The endpoint to use ( for instance ovh-eu)
+    application_key:
+        required: true
+        description:
+            - The applicationKey to use
+    application_secret:
+        required: true
+        description:
+            - The application secret to use
+    consumer_key:
+        required: true
+        description:
+            - The consumer key to use
+    timeout:
+        required: false
+        type: "int"
+        default: 120
+        description:
+            - The timeout in seconds used to wait for a task to be \
+            completed. Default is 120 seconds.
+
+'''
+
+EXAMPLES = '''
+# Adds or modify a backend to a loadbalancing
+- ovh_ip_loadbalancing name=ip-1.1.1.1 ip=212.1.1.1 state=present \
```

Comment:
ip=212.1.1.1 should be backend=212.1.1.1, no?
`unsupported parameter for module: ip`